### PR TITLE
D🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉C🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉m🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉M🐱‍🐉🐱‍🐉🐱‍🐉P🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉🐱‍🐉c🐱‍🐉h🐱‍🐉anges

### PR DESCRIPTION
**Pull request by SumedhP**

### CODE REVIEW

File: src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java [hunk 1]
```diff
@@ -117,7 +120,7 @@ public boolean hasTarget() {
      *         are degrees).
      */
     public double getVisionYaw() {
-        return shooterVisionSubsystem.getYaw();
+        return yawPass.calculate(shooterVisionSubsystem.getYaw()*0.5);
```
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java:123]: Why is this scaled by 1/2? (And why scale here and not in ShooterVisionSubsystem?)

File: src/main/java/frc/team2412/robot/Robot.java [hunk 2]
```diff
@@ -124,7 +125,7 @@ public void startCompetition() {
 
     @Override
     public void robotInit() {
-        LiveWindow.setEnabled(false);
+        // LiveWindow.setEnabled(false);
```
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/Robot.java:128]: Why was this commented out? Is this supposed to be permament? If so, please delete this and now-unused `LiveWindow` import.

File: src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java (hunk 1 above)
Comment (SumedhP) [reply src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java]: Was done to slow down turret before realizing the image was flipped, undoing

File: src/main/java/frc/team2412/robot/Robot.java (hunk 2 above)
Comment (SumedhP) [reply src/main/java/frc/team2412/robot/Robot.java:127]: It removed PDP logging, undid it cuz we wanted that

File: src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java [hunk 3]
```diff
@@ -36,14 +38,20 @@ public static class LocalizerConstants {
...
+    @Config.Exclude
     private final ShooterSubsystem shooterSubsystem;
-    @Exclude
+
+    @Log.Exclude
+    @Config.Exclude
     private final ShooterVisionSubsystem shooterVisionSubsystem;
 
     private final TimeBasedMedianFilter distanceFilter;
+    private final LinearFilter yawPass;
```
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java:54]: Is there a reason to used the time-based filter for distance and a moving average filter for yaw?

Comment (SumedhP) [reply src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java:54]: The distance filter is something we are going to test later in today's meeting

**Review by TAKBS2412 — APPROVED** [review]

**Comment by TAKBS2412** [comment]
Reminder so I don't forget to review this and #173 before leaving for worlds.

**Review by TAKBS2412 — DISMISSED** [review]

File: src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java [hunk 4]
```diff
@@ -377,12 +394,26 @@ public void updateModules(HolonomicDriveSignal driveSignal) {
...
+        }
+
         for (int i = 0; i < moduleOutputs.length; i++) {
             var module = modules[i];
             module.set(moduleOutputs[i].length * 12.0, moduleOutputs[i].getAngle().toRadians());
         }
     }
 
+    public void setToX() {
+        System.out.println("this is happening");
```
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java:410]: Should probably remove this

Comment (obj-obj) [reply src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java:410]: this is in fact happening

File: src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java [hunk 5]
```diff
@@ -311,6 +315,19 @@ public void resetPose(RigidTransform2 pose) {
         }
     }
 
+    @Config(tabName = "Drivebase", name = "reset modules", columnIndex = 5, rowIndex = 0)
```
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java:318]: We should make the subsystem consistent about using Oblog or direct NetworkTables, but I don't think that's worth delaying merging for.

**Review by KangarooKoala — DISMISSED** [review]

**Pull request merged**